manager/controllers/app/fybrik_notebook_read_flow_test.go

```diff
@@ -277,8 +278,18 @@ func TestS3NotebookReadFlow(t *testing.T) {
 	fmt.Println("Starting kubectl port-forward for arrow-flight")
 	portNum, err := strconv.Atoi(port)
 	g.Expect(err).To(gomega.BeNil())
-	listenPort, err := test.RunPortForward(modulesNamespace, svcName, portNum)
-	g.Expect(err).To(gomega.BeNil())
+
+	var listenPort string
+	i := 0
+	for true {
```

Comment:
This won't help. Kubectl port-forward is done once and not re-attempted. If you can undo this command upon failure, it would be better.